src/app/profile/page.tsx

```diff
@@ -51,18 +53,17 @@ export default function Page() {
         {profiles.map(profile => (
           <li key={profile.user_id}>
             <h2>Profile ID: {profile.user_id}</h2>
-            <p>First Name: {profile.first_name}</p>
```

Comment:
Can I ask what this function does, and why we are deleting the First Name and Last Name? 
Is it because we're now collecting First and Last Name in the Create Profile page?